dbbuild/sources/fan2022/Snakefile

```diff
@@ -0,0 +1,36 @@
+rule marker:
+    input:
+        script="collate.py",
+        tsv="marker-sans37.tsv",
+        bed37="marker37.bed",
+        bed38="marker38.bed",
+    output:
+        marker="marker.tsv"
+    shell:
+        "python {input} {output}"
+
+
+rule get_hg37_coords:
+    input:
+        bed38="marker38.bed",
+        chain="hg38ToHg19.over.chain.gz",
+    output:
+        bed37="marker37.bed",
+        bedunmapped="marker-unmapped.bed",
+    shell:
+        """
+        liftOver {input} {output}
+        [ ! -s {output.bedunmapped} ]
+        """
```

Comment:
Many of the SNPs/InDels don't have RSIDs, so positions for GRCh37 and GRCh38 must be provided explicitly. The authors provide only GRCh38 positions, so I whipped out UCSC's 38 --> 37 liftOver chain file for that conversion. Yay.

Reply:
oh fun!!